<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LowPassFilterSettings</class>
 <widget class="QWidget" name="LowPassFilterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="gb_lp_settings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Active</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sampling frequency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTraitDoubleSpinBox" name="sp_sampling_frequency">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>50000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTraitCheckBox" name="cb_overrule">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Low-pass frequency</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTraitDoubleSpinBox" name="sp_lowpass_frequency">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Order</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTraitSpinBox" name="sp_order">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTraitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>madernpytools/qtgui/traitlet_widgets</header>
  </customwidget>
  <customwidget>
   <class>QTraitSpinBox</class>
   <extends>QSpinBox</extends>
   <header>madernpytools/qtgui/traitlet_widgets</header>
  </customwidget>
  <customwidget>
   <class>QTraitCheckBox</class>
   <extends>QCheckBox</extends>
   <header>madernpytools/qtgui/traitlet_widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
